source-oracle/backfill.go

```diff
@@ -40,7 +40,7 @@ func (db *oracleDatabase) ScanTableChunk(ctx context.Context, info *sqlcapture.D
 		query = db.keylessScanQuery(info, schema, table)
 		args = []any{afterRowID}
 
-	case sqlcapture.TableStatePreciseBackfill:
+	case sqlcapture.TableStatePreciseBackfill, sqlcapture.TableStateUnfilteredBackfill:
```

Comment:
Following source-mysql to share the implementation between these two modes. This is necessary since sometimes `sqlcapture.TableStateUnfilteredBackfill` becomes the default mode for tables and users are confused when this happens and we complain about an invalid mode